benchmark/Project.toml

```diff
@@ -1,9 +1,9 @@
 [deps]
+ADTypes = "47edcb42-4c32-4615-8424-f2b9edc5f35b"
 BenchmarkTools = "6e4b80f9-dd63-53aa-95a3-0cdb28fa8baf"
 CSV = "336ed68f-0bac-5ca0-87d4-7b16caf5d00b"
 DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
 DifferentiationInterface = "a0c0ee7d-e4b9-4e03-894e-1c5f64a51d63"
-Diffractor = "9f5e2b26-1114-432f-b630-d3fe2085c51c"
```

Comment:
This is due to ADTypes not supporting Diffractor?

Reply:
And Diffractor was failing when I tried to call it through ChainRules. I have opened an issue on the repo, in the meantime I guess we can call it with the AbstractDiff interface as is recommended